CONTRIBUTION.md

```diff
@@ -15,18 +20,13 @@ Open your code editor of choice and edit your desired component/package in
 
 - We export only Web Components. No separate distribution of SASS mixins, JS, HTML and SCSS files.
 - We use [prettier](https://prettier.io/).
-  - Make sure you install eslint checker and prettier in your IDE
 
 ### Style Rules
 
 - We follow DRY, KISS, YAGNI principles when we write SCSS. This includes variables, mixins, etc.
 - We remove the linting of ordering and grouping of CSS properties. Only includes have to be at the beginning.
 - We won't have settings for vertical rhythm or global spacing.
 
-### JS Rules
```

Comment:
This is still valid rule, even with prettier

Reply:
but we removed some of them. the built ins being the most obvious ones.

Reply:
We removed one or two and Prettier overrides the formatting rules. Still, Eslint is for code style, prettier for formatting and settings are alomst same as plib 1

![Bildschirmfoto 2019-06-24 um 09 33 54](https://user-images.githubusercontent.com/275235/59999843-3c5a7280-9663-11e9-8c14-8c49dbd26eaa.png)

Reply:
@raphaellueckl maybe update the text like so:

We use the plibv1 eslint styling for code style, instead prettier takes care of all the formatting rules in the plib2

Reply:
I reverted it. I think it's better to go with the former approach imho, but we can also use your sentence. I think mentioning the linting rules here is enough.